docs/src/content/docs/es/guides/customization.mdx

```diff
@@ -86,6 +86,19 @@ Puedes mostrar diferentes versiones de tu logo en modos claro y oscuro.
    }),
    ```
 
+## Habilitar el mapa del sitio
+
+Starlight tiene soporte incorporado para generar un mapa del sitio. Habilita la generación del mapa del sitio estableciendo tu URL como `site` en `astro.config.mjs`:
```

Comment:
```suggestion
Starlight tiene soporte incorporado para generar un sitemap. Habilita la generación del sitemap estableciendo tu URL como `site` en `astro.config.mjs`:
```

Reply:
Hello Kevin, I've found that there is a proper translation in spanish for `sitemap` in spanish https://support.microsoft.com/es-es/office/generar-un-mapa-de-sitio-web-329d4803-f99a-4332-a4db-7ac3fd0d97da, should I roll back?

Reply:
In that case all good ✅